x-pack/plugin/core/src/test/java/org/elasticsearch/xpack/core/security/user/InternalUsersTests.java

```diff
@@ -215,13 +215,13 @@ public void testStorageUser() {
     }
 
     public void testDlmUser() {
-        assertThat(InternalUsers.getUser("_dlm"), is(InternalUsers.DLM_USER));
+        assertThat(InternalUsers.getUser("_dlm"), is(InternalUsers.DATA_STREAM_LIFECYCLE_USER));
```

Comment:
```suggestion
        assertThat(InternalUsers.getUser("_data_stream_lifecycle"), is(InternalUsers.DATA_STREAM_LIFECYCLE_USER));
```

Reply:
And two more spots in DataLifecycleServiceRuntimeSecurityIT: `unauthorized for user [_dlm]` -> `unauthorized for user [_data_stream_lifecycle]`

Reply:
Thank you! Fixed it.